src/sage/interfaces/maxima_abstract.py

```diff
@@ -1573,7 +1573,7 @@ def integral(self, var='x', min=None, max=None):
         ::
 
             sage: f = maxima('exp(x^2)').integral('x',0,1); f
-            -(sqrt(%pi)*%i*erf(%i))/2
+            -...(sqrt(%pi)*%i*erf(%i))/2...
```

Comment:
You can call `.sage()` on his one and compare it to the expected answer with `==`.